### DISCUSSION TITLE
Change the SSID for the NSPanel

### DISCUSSION THREAD

**Discussion by adamengineering** [Blackymas/NSPanel_HA_Blueprint#1797] [Q&A] [answered]
I've read the 'How To' part of the documentation here
[https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/main/docs/howto.md](url)

As per my understanding this is first of all, for changing the Password to the WiFi and the OTA.

My question is: **What do i need to take care of when I only want to change the SSID part?**

I've setup a new WLAN with a different SSID but with the same Password as before, so that I do have to change only parts of the various configurations for all my IOT's

**Comment by edwardtfn** [comment, marked as answer]
If the password isn’t changing, then you can just change the SSID in your substitutions and flash your panel (wirelessly is ok). It will restart after the ota update and then will try to connect to the new network.
